### DISCUSSION TITLE
Should the results of metaviralspades and metaplasmidspades be used with scaffolds from metaspades for binning?

### DISCUSSION THREAD

**Discussion by jolespin** [ablab/spades#1039] [Q&A]
My plan was to do the following:

* trim reads
* Input into metaplasmidspades, get reads that don’t map to scaffoldsA
* Input reads that don’t map to scaffoldsA into metaviralspades, get reads that don’t map to scaffoldsB
* Input reads that don’t map to scaffoldsB into metaspades
* combine all the scaffolds

this way, all the reads should only be used once and not contribute to multiple scaffolds in multiple algorithms.

My question is whether or not I should bin the metaviralspades scaffolds with the metaspades scaffolds for prokaryotic binning?  Should I keep those separate? I’m envisioning a case where a putative viral contig bins with a prokaryotic MAG and how to explain this.

**Comment by asl** [comment]
Well... your plan could lead to fragmented assemblies. Consider e.g. some common region between plasmid and chromosome. Or you can have some reads that would erroneously map to plasmid scaffolds, etc. So, in general your approach ignores repeats and would lead to suboptimal results.

**Comment by jolespin** [comment]
This is interesting, I hadn't thought of that.  I have a few followup questions: 
* Do you know how common it is for there to be strong homology between a plasmid and a chromosome? I'm sure this happens quite a bit because of HGT but I'm wondering if there are any studies that quantify this or show sequence bias in assemblies. 
* If the above is significant, what is the best use case for metaplasmidSPAdes? 
* Is the assembly process in biosyntheticSPAdes and metaplasmidSPAdes different than metaSPAdes or are contigs the same but classified as a BGC or plasmid? 
* Just to be clear, one should NOT use the same read set in one assembler as used in another (e.g., biosyntheticSPAdes and metaplasmidSPAdes and metaSPAdes) and we each read to only be incorporated into one contig from one assembler?
